samples/subsys/power/stm32_pm/src/main.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sample.h"
+
+struct device *gpio_bn, *gpio_led, *gpio_led2;
+
+/* in msec */
+#define WAIT_TIME 500
+#define SLEEP_TIME 4001
+/* setting to K_TICKS_FOREVER willl activate the deepsleep mode */
+
+#define DEMO_DESCRIPTION	\
+	"Demo Description\n"	\
+	"Application is demonstrating the LPTIMER used as systick.\n"\
+	"The system goes to sleep mode on k_sleep() and wakes-up\n" \
+	"(red led on) by the lptim timeout expiration\n" \
+	"or when pressing the user push button (WUP2 is PC13)\n\n"
```

Comment:
> WUP2 is PC13

Is this valid for all STM32 series ?

Reply:
most of the stm32 boards have a push button, if the soc has lptim